tests/framework/extensions/rke1/nodepools/nodepools.go

```diff
@@ -137,3 +140,82 @@ func ScaleNodePoolNodes(client *rancher.Client, cluster *management.Cluster, nod
 
 	return updatedNodePool, nil
 }
+
+// AddRKE1CustomClusterNodes is a helper method that will add nodes to the custom RKE1 custom cluster.
+func AddRKE1CustomClusterNodes(client *rancher.Client, cluster *management.Cluster, nodes []*nodes.Node, rolesPerNode []string) error {
+	token, err := tokenregistration.GetRegistrationToken(client, cluster.ID)
+	if err != nil {
+		return nil
+	}
+
+	var command string
+	for key, node := range nodes {
+		logrus.Infof("Adding node %s to cluster %s", node.NodeID, cluster.Name)
+		command = fmt.Sprintf("%s %s --address %s", token.NodeCommand, rolesPerNode[key], node.PublicIPAddress)
+
+		output, err := node.ExecuteCommand(command)
+		if err != nil {
+			return nil
+		}
+
+		logrus.Infof(output)
+	}
+
+	err = kwait.Poll(500*time.Millisecond, 10*time.Minute, func() (done bool, err error) {
+		clusterResp, err := client.Management.Cluster.ByID(cluster.ID)
+		if err != nil {
+			return false, err
+		}
+
+		if clusterResp.State == active && nodestat.AllManagementNodeReady(client, cluster.ID) == nil {
+			return true, nil
+		}
+		return false, nil
+	})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// DeleteRKE1CustomClusterNodes is a helper method that will delete nodes from the custom RKE1 custom cluster.
+func DeleteRKE1CustomClusterNodes(client *rancher.Client, cluster *management.Cluster, nodesToDelete []*nodes.Node) error {
+	nodes, err := client.Management.Node.ListAll(&types.ListOpts{Filters: map[string]interface{}{
```

Comment:
Same here Use the Steve v1 node call instead: https://github.com/rancher/rancher/blob/6376016a309a166c50fe6203b2e97dae0b90c89c/tests/framework/extensions/provisioning/verify.go#L454-L457. Let's limit our use of Norman use the corresponding Steve api call.

Reply:
The fix I had for RKE2/K3S didn't work well for RKE1. The annotation is different, but I accounted for that, but still no dice. I can keep investigating if you think we should match for RKE1.